src/vmm/src/cpu_config/x86_64/cpuid/normalize.rs

```diff
@@ -213,6 +225,21 @@ impl super::Cpuid {
         }
     }
 
+    /// Update vendor ID to prevent from vendor ID modification via custom CPU templates.
+    fn update_vendor_id(&mut self) -> Result<(), VendorIdError> {
```

Comment:
we probably need to mention it in the [doc](https://github.com/firecracker-microvm/firecracker/blob/main/docs/cpu_templates/cpuid-normalization.md).

Reply:
you're right! will do.